src/composables/networks.ts

```diff
@@ -142,31 +152,6 @@ export function useNetworks() {
     updateCustomNetwork,
     deleteCustomNetwork,
     resetNetworks,
-  };
-}
-
-/**
- * Monitor the network state and compare it with stored custom state to know when
- * user changes the network.
- */
-export function createNetworkWatcher() {
-  // Delay the networks fill in to allow the protocol adapters to be created first
-  setTimeout(() => {
-    ensureDefaultNetworksExists();
-  }, 0);
-
-  let storedNetworkName: string;
-
-  return {
-    onNetworkChange: (callback: (newNetwork: INetwork, oldNetwork: INetwork) => void) => {
-      setTimeout(() => {
```

Comment:
This `setTimeout` was one of the reasons of this PR. Now we don't need any 0s delays.